src/main/java/com/github/tomakehurst/wiremock/servlet/WireMockHttpServletRequestAdapter.java

```diff
@@ -172,6 +175,25 @@ public boolean isBrowserProxyRequest() {
         return false;
     }
 
+    private boolean isJettyRequest() {
+        if (!isPresent("org.mortbay.jetty.Request")) {
+            return false;
+        }
+        if (request instanceof org.mortbay.jetty.Request) {
+            return true;
+        }
+        return false;
+    }
+
+    private boolean isPresent(String className) {
+        try {
+            ClassUtils.getClass(className);
```

Comment:
Used commons-lang here (it was on the classpath anyway). I guess it could be done without that dependency if needed.